content/docs/user-guide/settings/settings-registry-how-to-dump-settings-registry.md

```diff
@@ -0,0 +1,272 @@
+---
+description: Describes how to dump an in-memory Settings Registry using in C++
+title: 'Output Settings Registry to Stream'
+linktitle: 'Output Settings Registry to Stream'
+weight: 400
+---
+
+- [Setting Registry Dump API](#setting-registry-dump-api)
+    - [SettingsRegistryMergeUtils DumpSettingsRegistryToStream() function](#settingsregistrymergeutils-dumpsettingsregistrytostream-function)
+        - [Example: Storing Editor Preferences using a key of "/Amazon/Editor/Preferences"](#dump-preferences-with-key)
+        - [Example: Storing Editor Preferences using a key of "/Amazon/Editor/Preferences" with an Anchor of "Amazon/Editor/Preferences"](#dump-preferences-with-key-and-anchor)
+        - [Example: Storing Editor Preferences using a key of "" and an include filter which maintains any JSON objects whose JSON pointer is a prefix of "/Amazon/Editor/Preferences"](#dump-preferences-with-key-and-include-filter)
+        - [Example: Saving the Editor Preferences section of the Setting Registry to a file](#saving-preferences-to-file)
+
+Setting Registry Dump API
```

Comment:
Sentence capitalization for docs. Would also change this page to Saving the Settings Registry so its clear what the point of doc is.